backend/src/api/candidates.js

```diff
@@ -296,16 +296,16 @@ router.put(
 )
 router.delete(
   '/:candidateId/interviews/:interviewId',
-  [leadsOnly],
+  [directorsOnly],
   errorWrap(async (req, res) => {
     let response = 'Interview Deleted Sucessfully'
     const candidate = await Candidate.findById(req.params.candidateId)
-    await candidate.interviews.id(req.params.interviewId).remove()
+    const result = await candidate.interviews.id(req.params.interviewId).remove()
```

Comment:
return what u deleted (should I not do this?)

Reply:
this is fine, I don't think we rely on DELETE endpoints returning the deleted objects anywhere, though that is a common pattern

Reply:
I think the standard practice is to just leave it out - the way to verify that a delete to occur would be to search in the database or see a UI refresh